docs/Concepts/Privacy/Private-Transactions.md

```diff
@@ -26,18 +26,18 @@ transaction itself.
 For more information about creating and sending private transactions, see the
 [How To documentation](../../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md).
 
-## Besu and Orion keys
+## Besu and Tessera keys
 
-Besu and Orion nodes both have public/private key pairs identifying them. A Besu node sending a
-private transaction to an Orion node signs the transaction with the Besu node private key. The
+Besu and Tessera nodes both have public/private key pairs identifying them. A Besu node sending a
+private transaction to an Tessera node signs the transaction with the Besu node private key. The
 `privateFrom` and `privateFor` attributes specified in the RLP-encoded transaction string for
 [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction) are the public
-keys of the Orion nodes sending and receiving the transaction.
+keys of the Tessera nodes sending and receiving the transaction.
 
 !!! important
 
-    The mapping of Besu node addresses to Orion node public keys is off-chain. That is, the sender
-    of a private transaction must know the Orion node public key of the recipient.
+    The mapping of Besu node addresses to Tessera node public keys is off-chain. That is, the sender
```

Comment:
Did we decide off-chain should be offchain?

Reply:
You right, should be "offchain".